tests/baselines/reference/parseErrorIncorrectReturnToken.js

```diff
@@ -17,9 +17,7 @@ let o = {
 string; // should be => not :
 // doesn't work in non-type contexts, where the return type is optional
 var f = function (n) { return function (string) { return n.toString(); }; };
-var o = {

```

Comment:
We didn't change the parser here, right? So why'd we lose the function in the output here?

Reply:
It was a parse error to begin with. Before the cleanup we would synthesize a body for a method without a body that wasn't an overload. As part of the cleanup we now always use the same code path to visit methods/functions even if the declaration itself doesn't have any changes. For correct code, nothing changes.

Reply:
Whereas now we elide it completely if there's no body and it has to have one? OK, I guess.